src/utils/utils.ts

```diff
@@ -3,6 +3,9 @@ import * as youtubedl from "youtube-dl";
 
 export const getInfo = Promise.promisify(youtubedl.getInfo);
 
+export const getInfoWithArg = Promise.promisify(
+    (url: string, arg: string[], cb: any) => youtubedl.getInfo(url, arg, cb));
```

Comment:
why callback is `any` not `(Error, Object) => void`?

Reply:
After promisify, the cb field should never be called in real application